.github/actions/check-previous-test-runs/action.yml

```
@@ -67,13 +67,23 @@ runs:
+      uses: peter-evans/find-comment@v3
+      id: find-comment
+      with:
+        issue-number: ${{ github.event.pull_request.number }}
+        comment-author: 'trezor-ci'
+        body-includes: ✅ Previously successful run of **${{ inputs.workflow_name }}**
+
+    - name: Post skipped tests comment
       if: ${{ steps.check-previous-runs.outputs.skip_tests == 'true' }}
       uses: peter-evans/create-or-update-comment@v4
       with:
+          comment-id: ${{ steps.find-comment.outputs.comment-id }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)